src/bci_build/package.py

```diff
@@ -1737,6 +1772,42 @@ def init_container_kwargs(os_version: OsVersion):
     for os_version in ALL_OS_VERSIONS
 ]
 
+REGISTRY_CONTAINERS = [
+    ApplicationStackContainer(
+        name="registry",
+        pretty_name="OCI Container Registry using the OCI Image Specification (Distribution)",
+        custom_description="Open Source Registry implementation using the OCI Distribution Specification",
+        package_name="distribution",
+        from_image=f"bci/bci-busybox:{OsContainer.version_to_container_os_version(os_version)}",
+        os_version=os_version,
+        is_latest=os_version in CAN_BE_LATEST_OS_VERSION,
+        version="%%registry_version%%",
+        version_in_uid=False,
+        replacements_via_service=[
+            Replacement(
+                regex_in_dockerfile="%%registry_version%%",
+                package_name="distribution-registry",
+                parse_version="minor",
+            )
+        ],
+        license="Apache-2.0",
+        package_list=[
+            Package(name, pkg_type=PackageType.BOOTSTRAP)
+            for name in (
+                "distribution-registry",
+                "apache2-utils",
+                "ca-certificates-mozilla",
+            )
+        ],
+        entrypoint=["/usr/bin/registry"],
+        cmd=["serve", "/etc/registry/config.yml"],
+        build_recipe_type=BuildType.KIWI,
+        volumes=["/var/lib/docker-registry"],
+        exposes_tcp=[5000],
+    )
+    for os_version in ALL_OS_VERSIONS
```

Comment:
Do you really want to create the container for 15 SP3 as well and not just SP4?

Reply:
changed to `CAN_BE_LATEST_OS_VERSION`